sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -761,6 +762,25 @@ dynamic parameter for the buffer can be registered with all the nodes which
 use the buffer as a parameter. Then a single `dynamic_parameter::update()` call
 will maintain the graphs data dependencies.
 
+===== Whole Graph Update [[whole-graph-update]]
+
+A graph in the executable state can have all of its nodes inputs & outputs
+updated using the `command_graph<graph_state::executable>::update(graph)`
+method. This method takes a graph in the modifiable state and updates the
+executable graph to use the node inputs & outputs of the modifiable graph.
+
+Both graphs must have been constructed with the same device and context.
```

Comment:
Should we clarify what "both" refers to here? Also, it could make sense to clarify what modifiable and executable mean regarding existing and resulting updated graph.

Reply:
> Also, it could make sense to clarify what modifiable and executable mean regarding existing and resulting updated graph.

I'm not quite sure what you mean by this, could you clarify?

Reply:
The first part of my comment is to clarify what "both graphs" refers to. In the second part, I was wondering if we want to introduce a language for the to be updated graph and the resulting updated graph. We could use these words then to clarify in which state each graph needs to be. I feel like we talk about modifiable and executable graph instead of their role in the update, which could be confusing.

Reply:
Is this not adequately described by the opening paragraph of this section? This explains that an executable graph can be updated using a modifiable graph.

I've added more clarity to `both graphs` but I'm not sure that further changes are necessary here.